src/benchmark_runner.cc

```diff
@@ -138,12 +142,16 @@ void RunInThread(const BenchmarkInstance* b, IterationCount iters,
 class BenchmarkRunner {
  public:
   BenchmarkRunner(const benchmark::internal::BenchmarkInstance& b_,
-                  std::vector<BenchmarkReporter::Run>* complexity_reports_)
+                  const size_t outer_repetitions_,
+                  const size_t inner_repetitions_,
+                  std::vector<BenchmarkReporter::Run>* complexity_reports_,
+                  RunResults* run_results_)
```

Comment:
if this is not expected to be null, we can take it as a pointer but store it as a reference (https://abseil.io/tips/116)

Reply:
DONE